Control feature for `tickets` appears broken

#### Description of problem:

When building products using control files with `tickets`, we get the following error:

```
RuntimeError: Unable to parse controls from /content/controls/cis_ocp_1_4_0.yml: Key tickets is not a valid for a control.
make[3]: *** [ocp4/CMakeFiles/ocp4-compile-all.dir/build.make:75: ocp4/ssg_build_compile_all-ocp4] Error 1
make[2]: *** [CMakeFiles/Makefile2:1954: ocp4/CMakeFiles/ocp4-compile-all.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2353: ocp4/CMakeFiles/generate-ssg-ocp4-ds.xml.dir/rule] Error 2
make: *** [Makefile:1071: generate-ssg-ocp4-ds.xml] Error 2 
```


#### SCAP Security Guide Version:


#### Operating System Version:


#### Steps to Reproduce:

1. Update a control file to use `tickets`
2. Run CI or attempt to build the profile

#### Actual Results:

```
RuntimeError: Unable to parse controls from /content/controls/cis_ocp_1_4_0.yml: Key tickets is not a valid for a control.
make[3]: *** [ocp4/CMakeFiles/ocp4-compile-all.dir/build.make:75: ocp4/ssg_build_compile_all-ocp4] Error 1
make[2]: *** [CMakeFiles/Makefile2:1954: ocp4/CMakeFiles/ocp4-compile-all.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2353: ocp4/CMakeFiles/generate-ssg-ocp4-ds.xml.dir/rule] Error 2
make: *** [Makefile:1071: generate-ssg-ocp4-ds.xml] Error 2 
```

#### Expected Results:

The content should build without issues, as documented in https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status. 

#### Additional Information/Debugging Steps:

--- a/ssg/controls.py
+++ b/ssg/controls.py
@@ -86,6 +86,10 @@ class Control(ssg.entities.common.SelectionHandler, ssg.entities.common.XCCDFEnt
         status_justification=str,
         fixtext=str,
         check=str,
+        tickets=list,
+        original_title=str,
+        related_rules=list,
+        controls=list,
     )
 
     MANDATORY_KEYS = {
@@ -108,6 +112,9 @@ class Control(ssg.entities.common.SelectionHandler, ssg.entities.common.XCCDFEnt
         self.fixtext = ""
         self.check = ""
         self.controls = []
+        self.tickets = []
+        self.original_title = ""
+        self.related_rules = []
 
     def __hash__(self):
         """ Controls are meant to be unique, so using the
@@ -117,9 +124,9 @@ class Control(ssg.entities.common.SelectionHandler, ssg.entities.common.XCCDFEnt
     @classmethod
     def _check_keys(cls, control_dict):
         for key in control_dict.keys():
-            if key not in cls.KEYS.keys() and key not in [
-                    'controls', 'original_title', 'related_rules', 'rules']:
-                raise ValueError("Key %s is not a valid for a control." % key)
+            # Rules shouldn't be in KEYS that data is in selections
+            if key not in cls.KEYS.keys() and key not in ['rules', ]:
+                raise ValueError("Key %s is not allowed in a control file." % key)
 
     @classmethod
     def from_control_dict(cls, control_dict, env_yaml=None, default_level=["default"]):
@@ -136,6 +143,10 @@ class Control(ssg.entities.common.SelectionHandler, ssg.entities.common.XCCDFEnt
         control.mitigation = control_dict.get('mitigation')
         control.fixtext = control_dict.get('fixtext')
         control.check = control_dict.get('check')
+        control.tickets = control_dict.get('tickets')
+        control.original_title = control_dict.get('original_title')
+        control.related_rules = control_dict.get('related_rules')
+
         if control.status == "automated":
             control.automated = "yes"
         if control.automated not in ["yes", "no", "partially"]:

